docker/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+# Check if /app/config_temp exists and is not empty
+if [ -d "/app/config_temp" ] && [ "$(ls -A /app/config_temp)" ]; then
+	# Copy from /config_temp to /config and remove /config_temp
+    cp -r /app/config_temp/* /app/config/
+    rm -rf /app/config_temp
+fi
+
+# Run
+exec java -jar /app/arrowhead-blacklist-5.0.0.jar -Dlog4j.configurationFile=/app/config/log4j2.xml
```

Comment:
need Xmx and Xms parameters, values should come from somewhere (environment variable?, arg?)
also version should come from somewhere

Reply:
Not needed as agreed with @rbocsi